message/messaging_service.hh

```diff
@@ -332,6 +332,7 @@ public:
     messaging_service(config cfg, scheduling_config scfg, std::shared_ptr<seastar::tls::credentials_builder>);
     ~messaging_service();
 
+    future<> init();
```

Comment:
Please, rename it to `start()` to be consistent with other services that need some future<>-returning initialization soon after construction with `sharded<>::start()`